workflow/schemas/config.schema.yaml

```diff
@@ -147,6 +147,17 @@ properties:
         type: string
         description: name or path to docker/singularity container
 
+  reference:
+    type: object
+    description: reference used by alignment
+    properties:
+      artifacts:
+        type: string
+        description: artifact file
```

Comment:
This description does not explain what kind of file it is and that it is a path. I think we should be specific, same for background

Reply:
@jonca79  could you give a suggestion for an appropriate text

Reply:
description: path to artifact file containing caller specific variant observation counts
description: path to background file containing median and sd values for non-reference bases per position

Reply:
thanks